release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -600,108 +600,219 @@ Some features available in previous releases have been deprecated or removed.
-|DEP
-|REM
-|REM
+|Removal of Jenkins images from install payload
+|General Availability
+|Removed
+|Removed
 
-|Bring your own {op-system-base} 7 compute machines
-|DEP
-|REM
-|REM
+|====
 
-|Jenkins Operator
-|DEP
-|REM
-|REM
+[discrete]
+=== Monitoring deprecated and removed features
+
+.Monitoring deprecated and removed tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.10 |4.11 | 4.12
 
 |Grafana component in monitoring stack
-|-
-|DEP
-|REM
+|Deprecated
+|Removed
+|Removed
 
 |Access to Prometheus and Grafana UIs in monitoring stack
-|
-|DEP
-|REM
+|Deprecated
+|Removed
+|Removed
+
+|====
+
+[discrete]
+=== Installation deprecated and removed features
+
+.Installation deprecated and removed tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.10 |4.11 | 4.12
 
 |vSphere 6.7 Update 2 or earlier
-|DEP
-|DEP
-|REM
+|Deprecated
+|Removed
+|Removed
 
 |vSphere 7.0 Update 1 or earlier
-|-
-|-
-|DEP
-
-|Virtual hardware version 13
-|DEP
-|DEP
-|REM
+|General Availability
+|Deprecated
+|Deprecated
 
 |VMware ESXi 6.7 Update 2 or earlier
-|DEP
-|DEP
-|REM
+|Deprecated
+|Removed
+|Removed
 
 |VMware ESXi 7.0 Update 1 or earlier
-|-
-|-
-|DEP
+|General Availability
+|Deprecated
+|Deprecated
 
-|Snapshot.storage.k8s.io/v1beta1 API endpoint
-|DEP
-|DEP
-|REM
+|CoreDNS wildcard queries for the `cluster.local` domain
+|General Availability
+|General Availability
+|Deprecated
 
-|Minting credentials for Microsoft Azure clusters
-|GA
-|REM
-|REM
+|====
+
+[discrete]
+=== Updating clusters deprecated and removed features
+
+.Updating clusters deprecated and removed tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.10 |4.11 | 4.12
+
+|Virtual hardware version 13
+|Deprecated
+|Removed
+|Removed
+
+|====
+
+[discrete]
+=== Storage deprecated and removed features
+
+.Storage deprecated and removed tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.10 |4.11 | 4.12
+
+|Snapshot.storage.k8s.io/v1beta1 API endpoint
+|Deprecated
+|Removed
+|Removed
 
 |Persistent storage using FlexVolume
-|-
-|DEP
-|DEP
+|Deprecated
+|Deprecated
+|Deprecated
+
+|====
+
+[discrete]
+=== Authentication and authorization deprecated and removed features
+
+.Authentication and authorization deprecated and removed tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.10 |4.11 | 4.12
 
 |Automatic generation of service account token secrets
-|GA
-|GA
-|REM
+|General Availability
+|Removed
+|Removed
 
-|Removal of Jenkins images from install payload
-|GA
-|GA
-|REM
+|====
+
+[discrete]
+=== Specialized hardware and driver enablement deprecated and removed features
+
+.Specialized hardware and driver enablement deprecated and removed tracker
+[cols="4,1,1,1",options="header"]
+|====
+|Feature |4.10 |4.11 | 4.12
+
+|Special Resource Operator (SRO)
+|General Availability
+|General Availability
+|Deprecated
+
+|====
+
+[discrete]
+=== Multi-architecture deprecated and removed features
+
+.Multi-architecture deprecated and removed tracker
```

Comment:
Should `ppc64le` and `s390x` be in backticks? (Not rhetorical, I don't know the answer to that :sweat_smile:)

Edit: reading further, I see that they are in backticks in the TP table, so seems like yes

Reply:
Thank you! I totally missed that. 😄